src/schema/broadband-signup-us.json

```diff
@@ -0,0 +1,559 @@
+        "property": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/Property",
+            "canonical": [
+                "/address/postcode",
+                "/address/city",
+                "/address/countrySubdivision"
+            ]
+        },
+        "selectedTvPackages": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/SelectedPackages",
+            "sourceOutputKey": "availableTvPackages",
+            "inputMethod": "SelectMany"
+        },
+        "selectedBroadbandPackage": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/SelectedPackage",
+            "sourceOutputKey": "availableBroadbandPackages",
+            "inputMethod": "SelectOne"
+        },
+        "selectedPhonePackage": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/SelectedPackage",
+            "sourceOutputKey": "availablePhonePackages",
+            "inputMethod": "SelectOne"
+        },
+        "contactPerson": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/PersonDob"
+        },
+        "account": {
+            "typeRef": "#/domains/Generic/types/Account"
+        },
+        "payment": {
+            "typeRef": "#/domains/Generic/types/Payment"
+        },
+        "panToken": {
+            "typeRef": "#/domains/Generic/types/PanToken"
+        },
+        "finalPriceConsent": {
+            "typeRef": "#/domains/Generic/types/PriceConsent",
+            "description": "Client's consent for final price, should exactly match the <code>finalPrice</code> object from output.<br/>Automation will not proceed with placing order until the consent is provided.",
+            "sourceOutputKey": "finalPrice",
+            "inputMethod": "Consent"
+        },
+        "landlineOptions": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/LandlineOptions"
+        },
+        "monthlyPaymentMethod": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/PaymentMethod"
+        },
+        "directDebit": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/DirectDebitIban"
+        },
+        "selectedTvSetupDate": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/SetupDate",
+            "sourceOutputKey": "availableTvSetupDates",
+            "inputMethod": "SelectOne"
+        },
+        "selectedBroadbandSetupDate": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/SetupDate",
+            "sourceOutputKey": "availableBroadbandSetupDates",
+            "inputMethod": "SelectOne"
+        },
+        "selectedActiveLandlineOption": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/String",
+            "sourceOutputKey": "availableActiveLandlineOptions",
+            "inputMethod": "SelectOne"
+        },
+        "selectedAddress": {
+            "typeRef": "#/domains/BroadbandSignupUS/types/PIIString",
+            "sourceOutputKey": "availableAddresses",
+            "inputMethod": "SelectOne"
+        },
+        "securityInformation": {
```

Comment:
Please rename this to `accountSecurity` if account related, or something else if related to something else.